.github/workflows/ci-server-testing.yml

```diff
@@ -0,0 +1,22 @@
+name: CI Server Testing and Client Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  run_tests:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - run: yarn install --frozen-lockfile
+      - run: yarn test
+        env:
+          MONGODB_CONNECTION: '${{ secrets.MONGODB_CONNECTION }}'
+      - run: cd client && yarn install && yarn build
```

Comment:
Should we also add a linting test? I actually don't think we have a linting script set up for our project (just a standalone eslint file)

Reply:
That would be a good idea, will make a ticket for that enhancement